pkg/compactor/meta_extenstions.go

```diff
@@ -0,0 +1,71 @@
+package compactor
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/thanos-io/thanos/pkg/block/metadata"
+)
+
+type CortexMetaExtensions struct {
```

Comment:
Can we move this file to `pkg/storage/tsdb`? So that it can be shared by other components